awx/ui_next/src/screens/Setting/SettingList.jsx

```diff
@@ -0,0 +1,169 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { withI18n } from '@lingui/react';
+import { t } from '@lingui/macro';
+import {
+  Card as _Card,
+  CardHeader as _CardHeader,
+  DataList,
+  DataListItem,
+  DataListCell,
+  DataListItemCells,
+  DataListItemRow,
+  PageSection as _PageSection,
+} from '@patternfly/react-core';
+import styled from 'styled-components';
+import { BrandName } from '../../variables';
+
+// Setting BrandName to a variable here is necessary to get the jest tests
+// passing.  Attempting to use BrandName in the template literal results
+// in failing tests.
+const brandName = BrandName;
+
+const PageSection = styled(_PageSection)`
+  column-count: 1;
+  column-gap: 24px;
+  @media (min-width: 576px) {
+    column-count: 2;
+  }
+`;
+const Card = styled(_Card)`
+  display: inline-block;
+  margin-bottom: 24px;
+  width: 100%;
+`;
+const CardHeader = styled(_CardHeader)`
+  display: inline-block;
+`;
+
+function SettingList({ i18n }) {
+  const settingRoutes = [
+    {
+      header: i18n._(t`Authentication`),
+      description: i18n._(
+        t`Enable simplified login for your Tower applications`
```

Comment:
@marshmalien We should use the `BrandName` [variable](https://github.com/ansible/awx/blob/devel/awx/ui_next/src/variables.js) for this to avoid putting a product name in the project

(or change it to "...for your applications" to keep it generic)